### DISCUSSION TITLE
ERROR! Failed to send check output to AWS Security Hub.

### DISCUSSION THREAD

**Discussion by cerontrustly** [prowler-cloud/prowler#1753] [Q&A] [answered]
Hello guys,

I'm using Prowler version 2.12.1 and recently integrated it with Security Hub. When running Prowler I can see this error being shown in some checks:

ERROR! Failed to send check output to AWS Security Hub.

I also noticed the checks reporting the above error are not being exported to Security Hub. My command line is:

docker run --rm --name prowler --env "AWS_ACCESS_KEY_ID" --env "AWS_SECRET_ACCESS_KEY" toniblyx/prowler:2.12.1 -r us-west-2 -f us-west-2 -g extras -M json-asff -S

Any ideias on what can be the issue?

Thank you!

**Comment by toniblyx** [comment]
Hi @cerontrustly, can you run it from the repo using `bash -x ./prowler <options>` with that you will get debug messages from the cli and we can figure out what is going on. Also feel free to use v3 (easier to install using pip) and it will take way less time than v2.

**Reply by cerontrustly** [reply]
Hi @toniblyx , thanks for helping! The message is always the same for every check that report that error message. Below some examples:

7.3 [extra73] Ensure there are no S3 buckets open to Everyone or Any AWS user - s3 [Critical]
 ERROR! Failed to send check output to AWS Security Hub.
       INFO! us-west-2: No buckets found

7.30 [extra730] Check if ACM Certificates are about to expire in 7 days or less - acm [High]
 ERROR! Failed to send check output to AWS Security Hub.
       INFO! us-west-2: No certificates found

Unfortunately I can't use v3 because the "extras" check group is missing...

**Comment by toniblyx** [comment, marked as answer]
You can do the same with prowler v3 if you locate all the extra checks from here  https://github.com/prowler-cloud/prowler/blob/prowler-2/groups/group7_extras and use the naming of v3 from here https://docs.prowler.cloud/en/latest/tutorials/aws/v2_to_v3_checks_mapping/ use a space separate list of checks in the cli.

**Reply by cerontrustly** [reply]
Thank you @toniblyx , that did the trick!

**Reply by toniblyx** [reply]
awesome!
